http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -669,6 +669,80 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
             .raiseError[Unit](AppNotFoundByWorkspaceIdException(workspaceId, appName, ctx.traceId, "permission denied"))
     } yield GetAppResponse.fromDbResult(app, Config.proxyConfig.proxyUrlBase)
 
+  // Autodeletion config logic requires a positive integer for threshold if enabled is true.
+  // For simplicity we can enforce this by rejecting the following scenarios:
+  // 1. The request includes a threshold which is invalid
```

Comment:
Can you define what an invalid threshold is here?

Reply:
updated